.github/ISSUE_TEMPLATE/2_bug_report_linux.md

```diff
@@ -36,9 +36,9 @@ differs from your own.
 
 ### Log file
 
-Make a log file made with -v -v or --log-file=output.txt, paste it to
-https://0x0.st/ or attach it to the github issue, and replace this text with a
-link to it.
+Make a log file made with --gpu-debug and -v -v or --log-file=output.txt,
```

Comment:
So you mean `--gpu-debug and (-v -v or --log-file=output.txt)` but it doesn't read like if you don't know it already.

How about:

`Make a log file with --gpu-debug, and either -v -v or --log-file=output.txt`